AcademicManagerV1/src/com/academic/db/dao/PersonDAO.java

```diff
@@ -8,22 +8,36 @@
 
 import com.academic.db.DAOImpl;
 import com.academic.model.Person;
-import com.mysql.jdbc.PreparedStatement;
+import java.sql.PreparedStatement;
 
 public class PersonDAO extends DAOImpl<Person> {
 	private PreparedStatement selectByIdStatement;
 	private PreparedStatement selectAllStatement;
 	private PreparedStatement countStatement;
+	//CRUD
+	private PreparedStatement addStatement;
+	private PreparedStatement updateStatement;
+	private PreparedStatement deleteStatement;
 
 	public PersonDAO(Connection conn) throws SQLException {
 		super(conn);
 		// TODO Auto-generated constructor stub
-		selectByIdStatement = (PreparedStatement) dbConnection.prepareStatement("SELECT personId,name,surname,type,dateOfBirth,email,phoneNumber,address,taxNumber,bankAccount,sex FROM person WHERE personId=?;",
+		selectByIdStatement = dbConnection.prepareStatement("SELECT personId,name,surname,type,dateOfBirth,email,phoneNumber,address,taxNumber,bankAccount,sex FROM person WHERE personId=?;",
 				ResultSet.TYPE_SCROLL_INSENSITIVE, ResultSet.CONCUR_READ_ONLY);
-		selectAllStatement = (PreparedStatement) dbConnection.prepareStatement("SELECT * FROM person;",
+		
+		selectAllStatement = dbConnection.prepareStatement("SELECT * FROM person;",
 				ResultSet.TYPE_SCROLL_INSENSITIVE, ResultSet.CONCUR_READ_ONLY);
-		countStatement = (PreparedStatement) dbConnection.prepareStatement("SELECT COUNT(*) FROM person;",
+		
+		countStatement = dbConnection.prepareStatement("SELECT COUNT(*) FROM person;",
 				ResultSet.TYPE_SCROLL_INSENSITIVE, ResultSet.CONCUR_READ_ONLY);
+
+		addStatement = dbConnection.prepareStatement(
+				"INSERT INTO person (personId,name,surname,type,dateOfBirth,email,phoneNumber,address,taxNumber,bankAccount,sex) VALUES (null, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?);");
+
+		updateStatement = dbConnection.prepareStatement(
+				"UPDATE person SET name=?,surname=?,dateOfBirth=?,email=?,phoneNumber=?,address=?,taxnumber=?,bankAccount=?,sex=? WHERE personId= ?;");
+		
+		deleteStatement = dbConnection.prepareStatement("DELETE * FROM person WHERE personId=?");
```

Comment:
Convert to soft delete (isDeleted=1). You will also need to make changes in the Database (?) and the other Select SQL statements